packages/mst-middlewares/src/redux.ts

```diff
@@ -80,7 +80,13 @@ function getActionContextNameAndTypePath(actionContext: ActionContext, logArgsNe
     let targetTypePath = actionContext.targetTypePath
 
     if (logArgsNearName) {
-        let args = actionContext.callArgs.map(a => JSON.stringify(a)).join(", ")
+        let args = actionContext.callArgs.map(a => {
+            try {
+                return JSON.stringify(a);
+            } catch (e) {
+                // Seems it may be a browser event?
```

Comment:
could you return here something like `**UNSERIALIZABLE**` so it doesn't leave an empty space?

Reply:
That should also fix the build error